pkgs/development/python-modules/google-auth-oauthlib/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, click
+, mock
+, pytest
+, futures
+, google_auth
+, requests_oauthlib
+}:
+
+buildPythonPackage rec {
+  pname = "google-auth-oauthlib";
+  version = "0.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "226d1d0960f86ba5d9efd426a70b291eaba96f47d071657e0254ea969025728a";
+  };
+
+  checkInputs = [
+    click mock pytest futures
+  ];
+
+  propagatedBuildInputs = [
+    google_auth requests_oauthlib
+  ];
+
+  checkPhase = ''
+    py.test
+  '';
+
+  meta = {
+    description = "Google Authentication Library: oauthlib integration";
+    homepage = https://github.com/GoogleCloudPlatform/google-auth-library-python-oauthlib;
+    license = lib.licenses.asl20;
+  };
```

Comment:
Don't you want to maintain this?

Reply:
I added myself as maintainer

Reply:
You should do so in a separate commit.

Reply:
Okay, I thought about that, but wasn't sure about the convention to name such commits. I will check the log.

Reply:
Done, sorry for the inconvenience, and thanks for the patience and keeping things tidy.